src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/utils/CypherUtils.java

```diff
@@ -50,4 +51,28 @@
         return records;
     }
 
+    public static boolean isPropertyColumn(final CypherQuery query, final CypherVar colName) {
```

Comment:
Please add a JavaDoc comment with a brief sentence about the purpose/functionality of this method.